File: oblig0/opg3/Person.java
class Person {
    private Bil3 bil;

    public Person(Bil3 privatbil) {
	bil = privatbil;
    }

    public void skrivBilNummer() {
	System.out.println("Personens bilnummer er " + bil.hentNummer() + ".");
    }
}

File: oblig0/opg3/c++/Person.cpp
#include "Person.h"
#include <iostream>

Person::Person(Bil3* bilen) : bil(std::move(bilen))
{
  std::cout << "Person created!" << std::endl;
}
Person::~Person()
{
  std::cout << "Person destructed!" << std::endl;
}
void Person::skrivBilNummer() {
  std::cout << "Personens bilnummer er " << bil->hentNummer() << "." << std::endl;
}
